docs/get-started/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -165,7 +166,33 @@ You can check it's working by using `search` subcommand and look for `ERROR` in
 ./quickwit index search --index hdfs-logs --query "severity_text:ERROR"
 ```
 
-Now that we have indexed the logs and can search from one instance, It's time to configure and start two other instances to form a cluster.
+which returns the json
+
+```json
+{
+  "num_hits": 345,
+  "hits": [
+    {
+      "attributes": {
+        "class": "org.apache.hadoop.hdfs.server.datanode.DataNode"
+      },
+      "body": "RECEIVED SIGNAL 15: SIGTERM",
+      "resource": {
+        "service": "datanode/16"
+      },
+      "severity_text": "ERROR",
+      "tenant_id": 51,
+      "timestamp": 1469687755
+    },
+    ...
+  ],
+  "elapsed_time_micros": 522542
+}
+```
+
+You can see that this query has 345 hits and that the server responds in 523 milliseconds.
```

Comment:
If you got `523 milliseconds` with a cold start, it's nice to add this information. Once the metastore is loaded, I would expect a lower response time.